src/App.tsx

```diff
@@ -1,7 +1,27 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyPressed);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyPressed);
+  }
+
+  handleKeyPressed = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">{this.state.pressedKey}</p>
```

Comment:
just make a condition
```suggestion
        <p className="App__message">
        {pressedKey ? 
          `The last pressed key is [${this.state.pressedKey}]` 
          : 'Nothing was pressed yet' }
        </p>
```